lib/Experian/IDAuth.pm

```diff
@@ -2,7 +2,7 @@ package Experian::IDAuth;
 use strict;
 use warnings;
 
-our $VERSION = '2.4';
```

Comment:
@raunakkathuria  then can I use `2.51` as the next version ? I think `2.x` will be `3.x` soon, too quickly.